### PR TITLE
Flesh out the FAQ content

### PR DESCRIPTION
**Pull request by benmccann**
Ref https://github.com/sveltejs/kit/issues/581. Not closing the issue because we still need the content surfaced on the site

I can't believe we already have over 100 issues :stuck_out_tongue_closed_eyes: Hoping this will help stem the tide somewhat, but that is probably wishful thinking

The last question has some sub-questions. I figured this was an okay way to start out and once the questions are surfaced on the site we can see how it renders and whether it should be adjusted a bit

### CODE REVIEW

File: documentation/faq/500-aliases.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+question: How do I setup a path alias?
+---
+
+Please be aware that you will probably want the alias specified in two places:
+* In `svelte.config.cjs` add [`vite.resolve.alias`](https://vitejs.dev/config/#resolve-alias)
+* In `tsconfig.json` (for TypeScript users) or `jsconfig.json` (for JavaScript users) to make VS Code aware of the alias
```
Comment (GrygrFlzr) [inline documentation/faq/500-aliases.md:7]: We might want to include an example:

```js
// svelte.config.cjs
const path = require('path');
module.exports = {
  kit: {
    vite: {
      resolve: {
        alias: {
          '$utils': path.resolve('./src/utils')
        }
      }
    }
  }
};
```
```jsonc
// tsconfig.json or jsconfig.json
{
  "compilerOptions": {
    "paths": {
      "$utils/*": ["src/utils/*"]
    }
  }
}
```

File: documentation/faq/700-integrations.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Please see [sveltejs/integrations](https://github.com/sveltejs/integrations#sveltekit) for examples of using many popular libraries like Tailwind, PostCSS, Firebase, GraphQL, mdsvex, and more.
+
+## How do I setup a database?
+
+Put the code to query your database in [endpoints](../docs#routing-endpoints) - don't query the database in .svelte files. You can create a `db.js` or similar that sets up a connection immediately and makes the client accessible throughout the app as a singleton. You can execute any one-time setup code in `hooks.js` and import your database helpers into any endpoint that needs them.
+
+## How do I use Axios?
+
+You probably don't need it. We'd generally recommend you just use `fetch` instead. If you insist, you're probably better off using an ESM-friendly alternative like `redaxios` [until axios utilizes ESM](https://github.com/axios/axios/issues/1879). Finally, if you still want to use Axios itself, try putting it in `optimizeDeps.include`.
```
Comment (GrygrFlzr) [inline documentation/faq/700-integrations.md:15]: "ESM-friendly alternative" sounds like it's a different API. Maybe "the ESM drop-in replacement `redaxios`" would be better, especially if you say you can just do `import axios from 'redaxios'` and get to work.

File: documentation/faq/700-integrations.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+question: How do I use X with SvelteKit?
+---
+
+## How do I setup library X?
+
+Please see [sveltejs/integrations](https://github.com/sveltejs/integrations#sveltekit) for examples of using many popular libraries like Tailwind, PostCSS, Firebase, GraphQL, mdsvex, and more.
```
Comment (dummdidumm) [inline documentation/faq/700-integrations.md:7]: We could also refer to the `svelte-adders` repo here

Comment (babichjacob) [reply documentation/faq/700-integrations.md:7]: It's actually within the integrations repo link (and is what the list of popular libraries based on).

Comment (dummdidumm) [reply documentation/faq/700-integrations.md:7]: Ah ok, nevermind then

File: documentation/faq/500-aliases.md (hunk 1 above)
Comment (benmccann) [reply documentation/faq/500-aliases.md]: done. thanks!

File: documentation/faq/700-integrations.md (hunk 2 above)
Comment (benmccann) [reply documentation/faq/700-integrations.md]: i like that. done

File: documentation/faq/70-integrations.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+question: How do I use X with SvelteKit?
+---
+
+## How do I setup library X?
```
Comment (pngwn) [inline documentation/faq/70-integrations.md:5]: Could you make these level 4 headings? I think will be the correct level for the site and it is easier to do it here than transforming the source.

Comment (benmccann) [reply documentation/faq/70-integrations.md]: done. I don't quite understand why though. Are there level 2 and 3 headings coming from somewhere?

Comment (pngwn) [reply documentation/faq/70-integrations.md]: Maybe level 3 is correct actually.

level 1 will be the page title which should be `svelte kit faq` or similar, level 2 will be the question itself (which is in the frontmatter), so maybe level 3 is correct for these subheadings.

Comment (pngwn) [reply documentation/faq/70-integrations.md]: I wrote this in chat, this is the issue:"

> The whole heading level thing for docs is a little bit annoying. I think we should probably just use relatively normal heading levels and transform them when we format the docs. That way the readmes at least make sense for readers looking at them in source and we are free to mess around with those levels at any point without needing to rewrite anything.

Comment (benmccann) [reply documentation/faq/70-integrations.md]: ok, updated to level 3

**Pull request merged**